.nvmrc

```diff
@@ -1 +1 @@
-18.18.2
+20.8.1
```

Comment:
Was going to point out that Node was bumped as part of [this PR](https://github.com/guardian/atom-workshop/pull/343/files#r1369797574), but I think maybe you forgot to push this, or pushed to this branch instead? Just highlighting in case there were any other associated changes that weren't pushed in that PR, that you may want to bring in here.

Might be worth noting this node upgrade in the PR description?

Reply:
Whoops, looks like I pushed that to the wrong branch. I've updated the other PR description and will mention here as you suggest. Good spot!